### ISSUE TITLE
Create chat rooms for sms thread - One room per phone number

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#33]
```
GTalkSMS create a chat room for all incoming SMS.
The room is the same for a contact.
No more command to send sms, just put text as a conversation.


```

Original issue reported on code.google.com by `Florent....@gmail.com` on 6 Jan 2011 at 11:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 11:16
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.7
```

Original comment by `Florent....@gmail.com` on 8 Jan 2011 at 11:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
